addons/test_website/static/tests/tours/custom_snippets.js

```diff
@@ -10,12 +10,13 @@ var tour = require('web_tour.tour');
  * -> drag a banner into page content
  * -> customize banner (set text)
  * -> save banner as custom snippet
- * -> confirm name (TODO: remove when implicit default name feature is implemented)
```

Comment:
This is a change in test_website and it is not mentioned in the commit message :wink: